starter_notebook_reverse_training.ipynb

```diff
@@ -0,0 +1,1292 @@
+{
```

Comment:
<p>A big visible comment would help to clarify the purpose of this notebook compared to the starter one. We can also rename this one as "into_English" for example, and the old starter notebook "from_English".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Noted will change that asap